scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/ParquetAvroIO.scala

```diff
@@ -159,7 +145,7 @@ object ParquetAvroIO {
   object ReadParam {
     val DefaultProjection: Schema = null
     val DefaultPredicate: FilterPredicate = null
-    val DefaultConfiguration: Configuration = null
+    def DefaultConfiguration: Configuration = ParquetConfiguration.empty()
```

Comment:
TBH, I'd prefer keeping this as null and handle it internally

Reply:
Agree, I think, partly because it matches other ScioIO param conventions and partly because I'm worried that over time we'll forget why this was made a `def` and refactor it back into a `val` :)